### PR TITLE
[DOCS] Welcome and Installation

### PR DESCRIPTION
**Pull request by shir22**
Docs update
Link to build:
https://drive.google.com/file/d/1cCng-ejRlPxc70r8JrRb4drP-dkaa1qi/view?usp=drive_link

### CODE REVIEW

File: docs/source/getting-started/installation.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+`the different tiers <https://deepchecks.com/pricing>`__ if you want 
+more info about the licensed versions, or contact us at info@deepchecks.com.
+
+There are two parts to the system:
+
+- The client side (``deepchecks-client`` Python package), for interacting with the 
+  monitoring app
+- The monitoring service itself (UI, backend, etc.), which is built of several 
+  docker containers and deployed with docker-compose.
+  You need to install it only if you're using the open-source self-hosted version.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:21]: can add link from "to install" to the oss deployment page

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+
+Installation Steps
+=====================
+
+In order to work with Deepchecks Monitoring, you need to:
+
+1. :ref:`Install the deepchecks-client with pip <installation__deepchecks_client>`, which installs the SDK for interacting
+   with the app, along with the :doc:`Deepchecks Testing <deepchecks:getting-started/welcome>` package.
+2. (Only for open source deployments) :ref:`Deploy the self-hosted open source <installation__deepchecks_self_hosted>`
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:34]: would set the text in the ( ) in bold, maybe move it to the end of the line?

File: docs/source/getting-started/installation.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+4. :ref:`Obtain your API key <installation__obtain_api_key>` from the app
+
+
+
+.. _installation__deepchecks_client:
+
+Step #1: Install ``deepchecks-client`` with pip
+====================================================
+
+deepchecks-client is a Python package that installs the SDK for interacting with the monitoring app and
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:47]:
```suggestion
deepchecks-client is a Python SDK package for interacting with the monitoring app. The package is based on the ...
```

File: docs/source/getting-started/installation.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+the :doc:`Deepchecks Testing <deepchecks:getting-started/welcome>` package, which can be useful when exploring existing issues
+(e.g. when using the "download" feature to further research the issue in your local environment)
+
+It can be installed using pip:
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:51]:
```suggestion
Both the SDK and Deepchecks Testing packages can be installed a single pip command:
```

File: docs/source/getting-started/installation.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+    pip install deepchecks-client --upgrade
+
+
+.. _installation__deepchecks_self_hosted:
+
+Step #2: Deploy the self-hosted open source
+==============================================
+
+If you're using the SaaS version, you can skip this step.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:63]: suggestion - put this in bold

File: docs/source/getting-started/installation.rst [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+.. _installation__create_or_join_organization:
+
+Step #3: Create or join an organization
+=============================================
+
+For SaaS usage: following an invite mail (sign up to our invite system at the `Deepchecks Website`_, or contact us at info@deepchecks.com),
+you'll be able to join a new organization.
+Then, head over to the deepchecks host URL (e.g. https://app.deepchecks.com or your unique url if received) to start exploring.
+
+For Oo-prem: create a user and organization and log in to the deepchecks UI, hosted at the url you've chosen during installation.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:77]:
```suggestion
For On-prem: create a user and organization and log in to the deepchecks UI, hosted at the url you've chosen during installation.
```

File: docs/source/getting-started/installation.rst [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+If you wish to self-host the deepchecks monitoring app, jump over to the :ref:`self-host deepchecks monitoring <installation__self_host_deepchecks>` installation instructions,
+and continue to the next step once you've completed the local installation and can open your local deepchecks monitoring instance.
+
+
+.. _installation__create_or_join_organization:
+
+Step #3: Create or join an organization
+=============================================
+
+For SaaS usage: following an invite mail (sign up to our invite system at the `Deepchecks Website`_, or contact us at info@deepchecks.com),
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:73]: In the whole doc the split is open-source / Saas . Would keep it here as well for consistency

File: docs/source/getting-started/installation.rst [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+We recommend seting an environment variable named ``DEEPCHECKS_API_TOKEN`` by running the following in your terminal:
+
+.. code-block:: bash
+
+    export DEEPCHECKS_API_TOKEN=replace-this-string-with-your-api-token
+
+If you need a new API key, the "regenerate" button will create and reveal a new key. This will invalidate the previously generated key.
+Then you'll be able to create a :class:`DeepchecksClient <deepchecks_client.DeepchecksClient>` and start interacting with the system.
+
+You can use the following code snippet to instantiate it and start sending data to the system.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:106]:
```suggestion
You can use the following code snippet to instantiate a :class:`DeepchecksClient <deepchecks_client.DeepchecksClient>` and start interacting with the system.
```

File: docs/source/getting-started/installation.rst [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+Note: save your API key as you will be able to view it only once.
+We recommend seting an environment variable named ``DEEPCHECKS_API_TOKEN`` by running the following in your terminal:
+
+.. code-block:: bash
+
+    export DEEPCHECKS_API_TOKEN=replace-this-string-with-your-api-token
+
+If you need a new API key, the "regenerate" button will create and reveal a new key. This will invalidate the previously generated key.
+Then you'll be able to create a :class:`DeepchecksClient <deepchecks_client.DeepchecksClient>` and start interacting with the system.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:104]:
```suggestion

```

File: docs/source/getting-started/installation.rst [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+
+You're ready to go!
+======================
+
+Congratulations, you're ready to start monitoring your models with deepchecks!
+
+Head over to our :doc:`Quickstart </user-guide/tabular/auto_quickstarts/plot_quickstart>`
+or see the full :doc:`User Guide </user-guide/index>`
+to get your model monitoring up and running.
```
Comment (Nadav-Barak) [inline docs/source/getting-started/installation.rst:126]:
```suggestion
to get your data & model monitoring up and running.
```

File: docs/source/getting-started/deploy_self_host_open_source.rst [hunk 11]
```diff
@@ -33,15 +33,24 @@ Requirements
...
 
 Installing
 ==========
 
-To get started, all you need to do is to run the following command, which will spin up a new Deepchecks deployment
+To get started: first make sure you have the docker engine up and running 
+(try running ``docker`` or ``docker ps`` in your cli and check if you receive relevant output).
+
+Now all you need to do is to run the following command, which will spin up a new Deepchecks deployment
 for us automatically!
```
Comment (Nadav-Barak) [inline docs/source/getting-started/deploy_self_host_open_source.rst:46]: Would move the "You will be asked for the" text here

**Review by Nadav-Barak — COMMENTED** [review]
Changes i couldn't add on git:
contributors-q-and-a channel -> contributors-q&a channel
"The installation will hang here a couple of minutes..." -> this need to stand out much more
Need to add a paragragh in the "installing" section on the open source deployment that demonstrate how can you verify your installation was successful and everything is ok

File: docs/source/getting-started/installation.rst (hunk 9 above)
Comment (shir22) [reply docs/source/getting-started/installation.rst:104]: why? relevant context, leaving it

**Review by Nadav-Barak — APPROVED** [review]
checkout the git secret CI tool, it is mad about something you added

**Pull request merged**